topics/statistics/tutorials/aberrant_pi3k_pathway_analysis/tutorial.md

```diff
@@ -0,0 +1,643 @@
+>    - \--genes: Comma separated string of HUGO symbols for target genes or targenes_list.csv file
+>    - \--diseases: Comma separated string of disease types/TCGA acronyms for classifier
+>    - \--folds: Number of cross validation folds
+>    - \--drop: Drop the input genes from the X matrix
+>    - \--seed: Value specifies the initial value of the random number seed
+>    - \--copy_number: Optional flag to supplement copy number to define Y
+>    - \--filter_count: Integer number for minimum count of mutations to include per disease
+>    - \--filter_prop: Floating value for minimum proportion of mutated samples per disease
+>    - \--num_features: Integer number for genes to include in classifier
+>    - \--alphas: Comma separated string of alphas to test in pipeline
+>    - \--l1_ratios: Comma separated string of l1 parameters to test
+>    - \--alt_genes: Comma separated string of alternative genes to test
+>    - \--alt_diseases: Comma separated string of alternative diseases to test
+>    - \--alt_filter_count: Integer number for minimum count of mutations to include per alt_diseases
+>    - \--alt_filter_prop: Floating value for minimum proportion of mutated samples alt_disease
+>    - \--alt_folder: String for location to save the classifier figures
+>    - \--remove_hyper: Store_true: remove hypermutated samples
+>    - \--keep_intermediate: Store_true: keep intermediate roc curve items
+>    - \--x_matrix: File name of features gene expression matrix to use
+>    - \--classifier_folder: String of the location to store classifier data
+{: .details}
+
+> ### {% icon comment %} Outputs
+> Several outputs are generated from this step: A combined disease model (Pan-model) and individual disease models are generated. Logistic regression with elastic net penalization outputs sparse classifiers with 150-250 genes whose scores or weights are important for classification tasks. This tool computes predictions, accuracy measurements and measures AUROC and AUPR curves for the combined disease model (Pan-model). Additionally, the output also includes above performance and prediction metrics when the pan-model applied to alternative genes and cancer types. An overall model statistics are provided in the form of classifier summary. Additionally, a tabular output that includes each gene mutation and copy number counts per each targene and proportions in various diseases is listed.
+>    - Log file for script run and additional information
+>    - alt_gene_alt_disease_summary.tsv
+>    - alt_summary_counts.csv
+>    - classifier_coefficients.tsv
+>    - classifier_summary.txt
+>    - pancan_roc_results.tsv
+>    - summary_counts.csv
+>    - cv_heatmap.pdf
+>    - Disease classifier figures: list of 2 files [disease_pr and disease_auroc]
+>    - all_disease_pr.pdf
+>    - all_disease_roc.pdf
+>    - alt_gene_alt_disease_aupr_bar.pdf
+>    - alt_gene_alt_disease_auroc_bar.pdf
+>    - disease_aupr.pdf
+>    - disease_auroc.pdf
```

Comment:
lot of outputs are generated here already, I don't know this tool, but if any of these are nice to examine at this point, it could be nice to discuss some of these outputs, or add a question here?

Reply:
@shiltemann It is a complex tool and many graphical outputs along with predictions and probabilities. I will add some discussion to these outputs.